app/assets/javascripts/data_sets/editTable.js.coffee

```diff
@@ -68,6 +156,18 @@ setupResizer = (grid) ->
 
   $(window).trigger 'resize'
 
+showError = (error) ->
+  $('.mainContent').children('.alert-danger').remove()
+
+  $('.mainContent').prepend """
+    <div class='alert alert-danger alert-dismissable'>
+      <button type='button' class='close' data-dismiss='alert' aria-hidden='true'>
```

Comment:
Another good candidate for handlebars (note for myself if I end up adding handlebars to more places).